llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -705,7 +707,30 @@ def fsdp_wrap_fn(self, module: nn.Module) -> bool:
 
     # Activation Checkpointing
     def activation_checkpointing_fn(self, module: nn.Module) -> bool:
-        return isinstance(module, MPTBlock)
+        act_ckpt_list = getattr(self.config, 'activation_checkpointing_target',
+                                None) or ['MPTBlock']
+
+        if 'MPTBlock' in act_ckpt_list or 'mptblock' in act_ckpt_list:
+            log.info(
```

Comment:
Can we make this log a bit more clear by only saying we are ignoring other subblock modules if we actually are (and specifying which ones we're ignoring)?